Websites/bugs.webkit.org/Bugzilla/BugUrl/Chromium.pm

```diff
@@ -22,28 +22,19 @@ use Bugzilla::Util;
 
 sub should_handle {
     my ($class, $uri) = @_;
-    return ($uri->authority =~ /^bugs.chromium.org$/i) ? 1 : 0;
+
+    # Chromium URLs have this form:
+    #   http(s)://issues.chromium.org/issues/1234
+    return (lc($uri->authority) eq 'issues.chromium.org'
+            and $uri->path =~ m|^/issues/\d+$| ? 1 : 0;
```

Comment:
Missing ")" at the end just before ";"

Reply:
Oops, that's what I get for editing using the GitHub web UI

Reply:
Note that https://github.com/WebKit/WebKit/blob/master/Websites/bugs.webkit.org/Bugzilla/BugUrl/Google.pm#L27 has the same issue currently.

Reply:
Sorry, last comment was mistaken, deleted it.

Reply:
But actually, looking at similar code in the other modules in https://github.com/WebKit/WebKit/tree/main/Websites/bugs.webkit.org/Bugzilla/BugUrl, it would be more consistent with the other code to add the closing parentheses before the `?` instead.

Reply:
You are right.